helm/cluster-api-provider-aws/templates/apps_v1_deployment_capa-controller-manager.yaml

```diff
@@ -50,7 +50,7 @@ spec:
         env:
         - name: AWS_SHARED_CREDENTIALS_FILE
           value: /home/.aws/credentials
-        image: '{{ .Values.infrastructure.image.name }}:{{ .Values.tag }}'
+        image: registry.k8s.io/cluster-api-aws/cluster-api-aws-controller:v1.5.0
```

Comment:
we definitely wanna use our retagged image, this sis wrong

Reply:
I just updated the `values.yaml` and ran `make generate`

Guess we have something wrong in the script

Reply:
oh, the domain has changed

Reply:
Fixed